index.d.ts

```diff
@@ -451,6 +451,13 @@ interface ConfigParams {
    * String value for the client's authentication method. Default is `none` when using response_type='id_token', otherwise `client_secret_basic`.
    */
   clientAuthMethod?: string;
+
+  /**
+   * Additional extra request body properties to be sent to the AS during code or refresh token exchange
+   */
+  tokenParameters?: {
```

Comment:
`tokenEndpointParams`

You also need to update the signature for `refresh` https://github.com/auth0/express-openid-connect/blob/b73edb95455ad6a04b51290f18eb71db54d068cd/index.d.ts#L633